store/localstore/local_version_provider.go

```diff
@@ -31,18 +32,26 @@ func (l *LocalVersionProvider) CurrentVersion() (kv.Version, error) {
 	l.mu.Lock()
 	defer l.mu.Unlock()
 
-	var ts uint64
-	ts = uint64((time.Now().UnixNano() / int64(time.Millisecond)) << timePrecisionOffset)
-	if l.lastTimestamp == uint64(ts) {
-		l.logical++
-		if l.logical >= 1<<timePrecisionOffset {
-			return kv.Version{}, ErrOverflow
+	for {
+		var ts uint64
+		ts = uint64((time.Now().UnixNano() / int64(time.Millisecond)) << timePrecisionOffset)
+
+		if l.lastTimestamp > ts {
```

Comment:
If l.lastTimestamp is one day ahead ts, tidb server will block for one day?
How about recommending dba restart tidb server if the time gap is too large?